src/ring_theory/power_series/basic.lean

```diff
@@ -501,9 +505,9 @@ mv_polynomial.ext _ _ $ λ m,
 begin
   rw [coeff_trunc, coeff_one],
   split_ifs with H H' H',
-  { subst m, erw mv_polynomial.coeff_C 0, simp },
-  { symmetry, erw mv_polynomial.coeff_monomial, convert if_neg (ne.elim (ne.symm H')), },
-  { symmetry, erw mv_polynomial.coeff_monomial, convert if_neg _,
+  { subst m, simp },
+  { symmetry, rw mv_polynomial.coeff_one, exact if_neg (ne.symm H'), },
+  { symmetry, rw mv_polynomial.coeff_one, refine if_neg _,
     intro H', apply H, subst m, intro s, exact nat.zero_le _ }
```

Comment:
```suggestion
    intro H', apply H, subst m, intro s, exact nat.zero_le _ }
```

Should 0 ≤ any finsupp not be its own lemma? (I don't know this part of the library too well)

Reply:
Your suggestion doesn't make any changes!

Reply:
It was just meant to be a way to highlight the line haha